README.md

```diff
@@ -230,3 +225,69 @@ function testHandler() {
   }
 }
 ```
+
+### Generating mock RPC handlers from fixtures
+
+The package also exports a getMockRpcHandlers util which can be useful for testing.
+Fixtures need to be of the following type
+
+```js
+type RpcResponse = Object | ResponseError;
+type RpcResponseMap = Array<{
+  args: Array<*>,
+  response: RpcResponse,
+}>;
+type RpcFixtureT = {[string]: RpcResponseMap | RpcResponse};
+```
+
+`getMockRpcHandlers` has the following interface:
+
+```js
+type getMockRpcHandlersT = (
+  fixtures: Array<RpcFixtureT>,
+  onMockRpc?: OnMockRpcCallbackT
+) => HandlerType;
+```
+
+For example:
+
+```js
+import {getMockRpcHandlers, ResponseError} from 'fusion-plugin-rpc';
+
+const rpcFixtures = {
+  GET_USER_SUCCESS: {
```

Comment:
I think this example is a bit confusing, since it shows more than the required API. I would recommend keeping the readme example as close to the API as possible. Could we just update rpcFixtures to be an array here and pass it directly into `getMockRpcHandlers`?

Reply:
will do

Reply:
problem with not having identifiers like `GET_USER_SUCCESS` is you cannot reuse fixtures
if we use arrays, devs need to refer a specific fixture using indices like `rpcFixtures[0]` etc